cpp/build-support/run_clang_format.py

```diff
@@ -96,6 +96,8 @@
             # Python 2 and Python 3.
             with open(filename, "rb") as reader:
                 # Run clang-format and capture its output
+                print([arguments.clang_format_binary,
```

Comment:
This was used for temporary debugging? You should remove it now :-)

Reply:
Yes, I should. I removed it.